packages/jest-reporters/package.json

```diff
@@ -26,6 +26,9 @@
     "source-map": "^0.6.0",
     "string-length": "^2.0.0"
   },
+  "peerDependencies": {
```

Comment:
normal dependency - you import from it. the other peer deps are since we just import types